features/steps/ios_steps.rb

```diff
@@ -30,10 +30,15 @@
 end
 
 When("I close the keyboard") do
-  steps %Q{
-    Given the element "close_keyboard" is present
-    And I click the element "close_keyboard"
-  }
+  case Maze.driver.capabilities["platformName"]
```

Comment:
I think checking `Maze.config.os` (== 'macos') would be preferable as it's explicitly passed on the CLI and not a product of the Appium connection. (Applies in various other places too)